samples/nrf9160/http_application_update/src/main.c

```diff
@@ -6,12 +6,15 @@
 #include <stdint.h>
```

Comment:
Move changes of sample into separate PR

Reply:
lets keep it, since its needed for modem upgrade support.

Have it in separate commit.